arkiverer/src/test/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/ApplicationTests.kt

```diff
@@ -478,7 +478,7 @@ class ApplicationTests {
 
 
 	private fun verifyDeleteRequestsToFilestorage(expectedCount: Int) {
-		verifyMockedDeleteRequests(expectedCount, appConfiguration.config.filestorageUrl.replace("?", "\\?") + ".*")
```

Comment:
The url used to be `/files?ids=1,2,3`, but is now `/files/1,2,3`. No more need to escape `?` in tests